conf/messages

```diff
@@ -129,8 +129,9 @@ generic.paye.error.pattern=Enter {1} PAYE reference in the correct format
 generic.paye.error.required=Enter {1} PAYE reference
 generic.ukEori.error.required=Enter {1} EORI number
 generic.ukEori.error.pattern=Enter {1} EORI number in the correct format
-generic.ukBankAccount.error.required=Enter a valid{1} account number
-generic.ukBankAccount.error.pattern=Enter a valid{1} account number like 12345678
+generic.ukBankAccount.error.required=Enter an account number
+generic.ukBankAccount.error.pattern=Account number must be 8 digits
+generic.ukBankAccount.error.pattern1=Enter a valid account number, like 12345678
```

Comment:
I think `pattern1` is not meaningful for this error. Shall we define `pattern`s like `.length` or `.valid`?

Reply:
updated the code